core/controllers/feedback_test.py

```diff
@@ -336,3 +344,243 @@ def _get_email(n):
             self.assertEqual(
                 response_dict['messages'][num + 1]['text'],
                 'New Message %s' % num)
+
+    def _return_true(self, thread_id, exploration_id):
+        return True
+
+    def _return_false(self, thread_id, exploration_id):
+        return False
+
+    def _return_null(self, param_one, param_two, param_three, param_four):
+        return None
+
+    def _return_null(self):
+        return None
+
+    def _create_suggestions(self):
```

Comment:
Perhaps better to create a new test suite (SuggestionsIntegrationTests) and have this be the setUp() method in it.